packages/query/package.json

```diff
@@ -40,7 +40,6 @@
     "@tanstack/react-query-persist-client": "4.35.7",
     "axios": "1.6.7",
     "bignumber.js": "9.1.2",
-    "react": "18.2.0",
```

Comment:
Glad you got it working 👍 Maybe we can standardise the react version across everywhere we need it? 

I'm not sure why the extension would have a different version than other packages.

We could enforce react version and node version just to make life easier?

Reply:
Was think we could just update react-query to current version, but doesn't seem like the issue.